inst/rmd/h/anchor-sections/anchor-sections.css

```
@@ -0,0 +1,3 @@
+/* Styles for section anchors */
+a.anchor-section {margin-left: 10px; visibility: hidden; color: inherit;}
+.hasAnchor:hover a.anchor-section::before {visibility: visible; content: '#';}
```

I think you need to separate `visibility` and `content` into two CSS rules:

```suggestion
a.anchor-section::before {content: '#';}
.hasAnchor:hover a.anchor-section {visibility: visible;}
```

---

Oh really? At least it works fine with the reprex above.
https://github.com/rstudio/rmarkdown/pull/1884#issuecomment-703099032

But I'm fine with the suggested change.

---

Ah I see. Your suggestion works better when users specify custom CSS (e.g., `::after`)

---

Yeah that was just a very minor suggestion. I wanted the whole `<a>` to be visible on hover in case it contains anything else (such as `::after`). I'm okay no matter if you want to make this change or not :)